lib/GLPI/Agent/Task/Inventory/MacOS/AntiVirus/SentinelOne.pm

```diff
@@ -0,0 +1,76 @@
+package GLPI::Agent::Task::Inventory::MacOS::AntiVirus::SentinelOne;
+
+use strict;
+use warnings;
+
+use parent 'GLPI::Agent::Task::Inventory::Module';
+
+use GLPI::Agent::Tools;
+
+my $command = '/Library/Sentinel/sentinel-agent.bundle/Contents/MacOS/sentinelctl';
+
+sub isEnabled {
+    return canRun($command);
+}
+
+sub doInventory {
+    my (%params) = @_;
+
+    my $inventory = $params{inventory};
+    my $logger    = $params{logger};
+
+    my $antivirus = _getSentinelOne(logger => $logger);
+    if ($antivirus) {
+        $inventory->addEntry(
+            section => 'ANTIVIRUS',
+            entry   => $antivirus
+        );
+
+        $logger->debug2("Added $antivirus->{NAME} ".($antivirus->{VERSION}? " v$antivirus->{VERSION}":""))
+            if $logger;
+    }
+}
+
+sub _getSentinelOne {
+    my (%params) = @_;
+
+    my $antivirus = {
+        COMPANY     => "Sentinel Labs Inc.",
+        NAME        => "SentinelOne EPP",
+        ENABLED     => 0,
+    };
+
+    # Support file case for unittests if basefile is provided
+    if (empty($params{basefile})) {
+        $params{command} = "\"$command\" version";
+    } else {
+        $params{file} = $params{basefile}."-version";
+    }
+    my $version = getFirstMatch(
+        pattern => qr/^SentinelOne.* ([0-9.]+)$/,
+        %params
+    );
+    $antivirus->{VERSION} = $version if $version;
+
+    # Support file case for unittests if basefile is provided
+    if (empty($params{basefile})) {
+        $params{command} = "\"$command\" status";
+    } else {
+        $params{file} = $params{basefile}."-status";
+    }
+    my $base_version = getFirstMatch(
+        pattern => qr/^\s.*staticSignatures:\s.*(\([0-9]+\))/i,
+        %params
+    );
+    $antivirus->{BASE_VERSION} = $base_version if $base_version;
+
+    my $status = getFirstMatch(
+        pattern => qr/^\s.*Protection.*(enabled)$/i,
+        %params
+    );
+    $antivirus->{ENABLED} = 1 if $status && $status =~ /^enabled$/i; 
```

Comment:
Firstly, not a big deal here, but you code involves to start the command 2 times with the same argument. This should be optimized.

Secondly, it seems to me the used regexp patterns are not correct. And even the test is overkill on the "ENABLED" check. For the first pattern, I think it should be better this one: `qr/^\s+staticSignatures:.*(\([0-9]+\))/i`. For the second, you should only use a match on: `/^\s+Protection:\s+(enabled)$/i`.

This can give something like:
```suggestion
    my @lines = getAllLines(%params);
    my $base_version_line = first { /^\s+staticSignatures:\s/i } @lines;
    $antivirus->{BASE_VERSION} = $1
        if $base_version_line && $base_version_line =~ /^\s+staticSignatures:.*(\([0-9]+\))/i;

    my $status = getFirstMatch(
        pattern => qr/^\s.*Protection.*(enabled)$/i,
        %params
    );
    $antivirus->{ENABLED} = 1 if first { /^\s+Protection:\s+enabled$/i } @lines; 
```